Sources/PaymentsCore/Networking/APIClient.swift

```diff
@@ -24,55 +24,48 @@ public final class APIClient {
         }
 
         let task = urlSession.dataTask(with: request) { data, response, error in
-            let finish: (Result<T.ResponseType, NetworkingError>) -> Void = { result in
```

Comment:
Did some cleanup here to make this easier to read. 

_If this is too much/confusing for this PR, we can make a separate PR._

Reply:
thanks Sammy!! I don't think it's to much.
Although probably I'd move `correlationID` to the top since no matter whether it's error or not, as long as we have it, we should pass it to merchants